docs/source/installation.rst

```diff
@@ -22,8 +22,8 @@ Generic requirements
   faster than python version of the client, but still slower than C version.
 * ``psutil`` python package. Required for some segments like cpu_percent. Some 
   segments have linux-only fallbacks for ``psutil`` functionality.
-* ``mercurial`` python package (note: *not* standalone executable). Required to 
-  work with mercurial repositories.
+* ``hglib`` python package (note: *not* just standalone mercurial executable).
```

Comment:
This should be something like “`hglib` python package and mercurial executable”, `hglib` did not list mercurial python package as its dependency.